fw/http_parser.c

```diff
@@ -4825,9 +4826,10 @@ tfw_http_init_parser_req(TfwHttpReq *req)
 	 * - raw:
 	 *     none;
 	 * - spec:
-	 *     Connection: RFC 7230 6.1.
+	 *     Connection, Keep-alive: RFC 9110 7.6.1.
 	 */
-	hbh_hdrs->spec = 0x1 << TFW_HTTP_HDR_CONNECTION;
+	hbh_hdrs->spec = 0x1 << TFW_HTTP_HDR_CONNECTION |
```

Comment:
According https://www.rfc-editor.org/rfc/rfc2068#section-13.5.1:
   The following HTTP/1.1 headers are hop-by-hop headers:
     o  Connection
     o  Keep-Alive
     o  Public
     o  Proxy-Authenticate
     o  Transfer-Encoding
     o  Upgrade
Why we don't mark them all?